include/uv.h

```diff
@@ -1668,6 +1668,7 @@ union uv_any_req {
 };
 #undef XX
 
+typedef void (*uv_loop_cb)(uv_loop_t*);
```

Comment:
can we give this a better name it reads like "loop callback" but we might introduce more in the future.